doc/classes/WorkerThreadPool.xml

```diff
@@ -58,6 +58,7 @@
 				Adds [param action] as a group task to be executed by the worker threads. The [Callable] will be called a number of times based on [param elements], with the first thread calling it with the value [code]0[/code] as a parameter, and each consecutive execution incrementing this value by 1 until it reaches [code]element - 1[/code].
 				The number of threads the task is distributed to is defined by [param tasks_needed], where the default value [code]-1[/code] means it is distributed to all worker threads. [param high_priority] determines if the task has a high priority or a low priority (default). You can optionally provide a [param description] to help with debugging.
 				Returns a group task ID that can be used by other methods.
+				[b]Note:[/b] Every task must be waited for completion at some point so that any allocated resources inside the task can be cleaned up.
```

Comment:
```suggestion
				[b]Warning:[/b] Every task must be waited for completion at some point so that any allocated resources inside the task can be cleaned up.
```
I think this warrants a "warning", same below

Reply:
Thansk!